include/sensors/firmware/pressure_sensor_callbacks.h

```diff
@@ -0,0 +1,11 @@
+#include "platform_specific_hal_conf.h"
+#include "sensors/core/tasks/pressure_driver.hpp"
+
```

Comment:
Still not great at lifetimes of things. @sfoster1 is it better to have a global variable which the callback gets saved to, or is it fine to have `drdy_interrupt_response` be a static function on the pressure driver class?